projects/redis.io/package.yml

```diff
@@ -0,0 +1,31 @@
+distributable:
+  url: https://download.redis.io/releases/redis-{{ version }}.tar.gz
+  strip-components: 1
+
+versions:
+  github: redis/redis
+
+
+provides:
+  - bin/redis-server
+  - bin/redis-cli
+  - bin/redis-benchmark
+
+dependencies:
+  openssl.org: ^1
+
+build:
+  dependencies:
+    gnu.org/make: '>=4.3'
```

Comment:
Does it require GMake specifically, or is `tea.xyz/gx/make: '*'` sufficient?

Reply:
tested replacement and it works